contracts/DomainRegistry.sol

```diff
@@ -1,25 +1,44 @@
 //SPDX-License-Identifier: MIT
-pragma solidity ^0.8.17;
+pragma solidity ^0.8.20;
 
+import "@openzeppelin/contracts/utils/structs/EnumerableMap.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
 import "./DomainParserLibrary.sol";
 
-uint256 constant DEPOSIT_PRICE = 1 ether;
+contract DomainRegistry is Ownable {
+    using EnumerableMap for EnumerableMap.UintToUintMap;
+
+    EnumerableMap.UintToUintMap private domainLevelPrices;
```

Comment:
Why do you need enumerable map? It takes more gas then regular mapping. I don't see any strong reason for that. At least you didn't introduce any public methods which enumerate through current level prices.
   -1 point

Reply:
fixed in https://github.com/vpmoon/blockchain-playground/pull/5/commits/0218c5fe560e96a028b1ff0de3f4b8204409d2e4